bigtable-hbase/src/test/java/com/google/cloud/bigtable/hbase/adapters/TestColumnDescriptorAdapter.java

```diff
@@ -191,7 +191,10 @@ public void badExpressionLessThan(){
 
   @Test
   public void testBlankExpression(){
-    HColumnDescriptor result = adapter.adapt("family", asColumnFamily(""));
-    Assert.assertEquals(new HColumnDescriptor("family"), result);
+    // An empty gc expression means keep all versions, which is not supported by HBase.
+    expectedException.expectMessage("Maximum versions must be positive");
+    expectedException.expect(IllegalArgumentException.class);
```

Comment:
Apparently HBase doesn't support it, so we can't pipe things through.
https://github.com/apache/hbase/blob/master/hbase-client/src/main/java/org/apache/hadoop/hbase/HColumnDescriptor.java#L507